docs/documentation/e2e.md

```diff
@@ -37,17 +37,28 @@ End-to-end tests are run via [Protractor] (https://angular.github.io/protractor/
 <details>
   <summary>serve</summary>
   <p>
-    `--serve` (alias: `-s`) _default value: true_
+    <code>--serve</code> (aliases: <code>-s</code>) <em>default value: true</em>
   </p>
   <p>
     Compile and Serve the app. All serve options are also available. The live-reload option defaults to false, and the default port will be random.
   </p>
 </details>
 
 <details>
+  <summary>no-serve</summary>
```

Comment:
The 'no-' prefix can be used with any Boolean option.  A general statement on each command page would probably be more appropriate than singling out the serve option.